### ISSUE TITLE
IntIntMap is broken - containsKey returns true if map is empty

### ISSUE THREAD

**Issue by playmyskay** [libgdx/libgdx#5919]
**descpription**
Commit a4646767699501745cdd44ae0e5946d9296a2ce6 has negatively affected the functionality. 
An empty IntIntMap instance returns true if you call method _containsKey_.

**example code**
```java
IntIntMap map = new IntIntMap();
System.out.println(map.containsKey(1)); // output: true
```

**version of LibGDX**
1.9.11-SNAPSHOT

**affected platforms**
probably all?

- [ ] Android
- [ ] iOS (robovm)
- [ ] iOS (MOE)
- [ ] HTML/GWT
- [ ] Windows
- [x] Linux
- [ ] MacOS

**Comment by tommyettinger** [comment]
On it. No idea what could cause this, yet; an empty IntIntMap has about 64 capacity, so this doesn't seem related to the bug that was fixed over the last PR round with initial capacity 0. I'll see what reproduces this first; what data structures and what keys.

**Comment by tommyettinger** [comment]
I left comments where I saw flipped conditions in that recent commit. It looks like fixing this should be very quick; this is so small and so clear that it should not require another dreadful back-and-forth over a PR. I can't leave a comment on the LongMap changes; it may be affected, but the Object-keyed maps seem to be correct already.

**Comment by NathanSweet** [comment]
Fixed in a03b22b66300d15ae4fd1fae63a785fef853f6ac, sorry for the trouble.
